discovery-infra/test_infra/utils.py

```diff
@@ -346,6 +346,91 @@ def is_cluster_in_status(client, cluster_id, statuses):
         log.exception("Failed to get cluster %s info", cluster_id)
 
 
+def wait_till_cluster_validation_is_in_status(
+    client, cluster_id, validation_section, validation_id, statuses, timeout=consts.VALIDATION_TIMEOUT, interval=2
+):
+    log.info("Wait till cluster %s validation %s is in status %s", cluster_id, validation_id, statuses)
+    try:
+        waiting.wait(
+            lambda: is_cluster_validation_in_status(
+                client=client,
+                cluster_id=cluster_id,
+                validation_section=validation_section,
+                validation_id=validation_id,
+                statuses=statuses
+            ),
+            timeout_seconds=timeout,
+            sleep_seconds=interval,
+            waiting_for="Cluster validation to be in status %s" % statuses,
+        )
+    except:
+        log.error("Cluster validation status is: %s", get_cluster_validation_value(client.cluster_get(cluster_id), validation_section, validation_id))
```

Comment:
Python PEP8 recommend no more then 80 in one line, try to break the line whenever needed. top 120 is OK

Reply:
Will change, but if you want to adhere to pep8 it should run as part of the make process

Reply:
it is just that the line is too long and less convenient to read via github and on editor